test/system/252-quadlet.bats

```diff
@@ -976,9 +976,16 @@ EOF
 
       service_setup $QUADLET_SERVICE_NAME
 
-      # FIXME: log.91: Starting, not Started
       # Ensure we have output. Output is synced via sd-notify (socat in Exec)
-      run journalctl "--since=$STARTED_TIME" --unit="$QUADLET_SERVICE_NAME"
+      # When running under heavy load (e.g. parallel tests), it
+      # may take a little while for service to reach Started
+      for tries in $(seq 1 5); do
+          run journalctl "--since=$STARTED_TIME" --unit="$QUADLET_SERVICE_NAME"
```

Comment:
AFAIK systemctl should wait for it to reach started state, but we know journald is not updating fast enough even after you write to it successfully (i.e. all the races in podman logs with journald) so there sadly doesn't seem to be any way avoid these loops